tensorflow_addons/rnn/tests/cell_test.py

```diff
@@ -558,3 +558,64 @@ def test_esn_config():
     restored_cell = rnn_cell.ESNCell.from_config(config)
     restored_config = restored_cell.get_config()
     assert config == restored_config
+
+
+def test_zoneout():
```

Comment:
Please, add another test case for an e2e test with keras model and compile/fit. You can take `test_esn_keras_rnn_e2e` as an example.